cli.js

```diff
@@ -3,6 +3,7 @@
 const meow = require('meow');
 const getStdin = require('get-stdin');
 const boxen = require('boxen');
+const leftPad = require('left-pad');
```

Comment:
Sure, but I don't want the risk or condonement of such module. `indent-string` is a better fit for this use-case anyways.